base/bootstrap.go

```diff
@@ -59,6 +64,41 @@ func NewCouchbaseCluster(server, username, password,
 	return &CouchbaseCluster{c: cluster}, nil
 }
 
+func (cc *CouchbaseCluster) GetConfigLocations() ([]string, error) {
+	if cc == nil {
+		return nil, errors.New("nil CouchbaseCluster")
+	}
+
+	buckets, err := cc.c.Buckets().GetAllBuckets(nil)
+	if err != nil {
+		return nil, err
+	}
+
+	bucketList := make([]string, len(buckets))
+	for bucketName := range buckets {
+		bucketList = append(bucketList, bucketName)
+	}
+
+	return bucketList, nil
+}
+
+func (cc *CouchbaseCluster) GetConfig(location, groupID string, valuePtr interface{}) (cas uint64, err error) {
```

Comment:
In the current implementation location is just bucketName.  Might be clearer to leave that explicit here for now, as this function will need rework anyway if location is changed to [bucket,scope,collection].

Reply:
Makes sense. I've just renamed all references to "location" to be "bucket" until we're ready to support collections.